melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Vedtak.java

```diff
@@ -5,6 +5,6 @@
 @Data
 public abstract class Vedtak {
     private String datoforrigevedtak;
-    private String eropprinneligvedtak;
-    private String erendringsvedtak;
+    private String eropprinneligvedtak; // Pga. RINA regler: Kan bare sette "ja" eller null (default: nei)
+    private String erendringsvedtak; // Pga. RINA regler: Kan bare sette "nei" eller null (default: ja)
```

Comment:
Bra med kommentar her!
Man kan vel bare sette "nei", "ja" eller null?. Og antar at null er det samme som "nei"?

Reply:
Nei, kan ikke sette noe annet på "erendringsvedtak" enn "nei" eller null. Kan ikke sette ja her.
Det samme gjelder for "eropprinneligvedtak" med at vi bare kan sette "ja" eller null. Kan ikke sette nei her.

Reply:
Men på `eropprinneligvedtak` har du kommentert 
`// Pga. RINA regler: Kan bare sette "ja" eller null (default: nei)`
Om default er `nei` må vel det også kunne settes? Så burde vel stå 
`// Pga. RINA regler: Kan bare sette "ja", "nei" eller null (default: nei)` ?

Reply:
Nei, fordi det er feil. 
null == "nei" 

Vi godtar bare "ja" eller null, der null er det motsatte av "ja".
Har du forslag til kommentar som gjør det lettere å forstå?